homework-18/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Find your movie</title>
+</head>
+
+<body>
+  <div class="container">
+    <h1 class="header">Find Your Movie</h1>
+    <label for="input">

```

Comment:
It should have a label itself. If you don't want to show it to users then use some CSS classes to hide it. Having a label is a required thing in accessibility concerns

```suggestion
    <label for="input">Text here</label>
```